zeus/queue/base.py

```diff
@@ -0,0 +1,93 @@
+from typing import Callable, List, Type
+
+from .scheduler import Scheduler
+
+
+class UnknownTask(Exception):
+    pass
+
+
+def generate_task_name(func):
+    return "{}.{}".format(func.__module__, func.__name__)
+
+
+def run_task(task_name, task_kwargs):
```

Comment:
this was gross but was the easiest way i could come up with to make ``enqueue()`` work on both adapters (since both adapters expect you to pass a function to execute)